### PR TITLE
[12.0][MIG] Migrate project_task_category from 10.0 to 12.0

### PR DESCRIPTION
**Pull request by ruter-lyu**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please migrate the more general module `project_category` from v11.

**Comment by pedrobaeza** [comment]
In fact, it's already done in #470

File: project_task_category/README.rst [hunk 1]
```diff
@@ -0,0 +1,42 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
```
Comment (nikul-serpentcs) [inline project_task_category/README.rst]: @ruter-lyu LGPL-3.0

File: project_task_category/data/project_data.xml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+© 2017-2019 Elico Corp (https://www.elico-corp.com).
+License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+-->
+<odoo>
```
Comment (nikul-serpentcs) [inline project_task_category/data/project_data.xml]: `<odoo noupdate="1">`

**Review by nikul-serpentcs — CHANGES_REQUESTED** [review]
Minor Change

File: project_task_category/README.rst (hunk 1 above)
Comment (nikul-serpentcs) [inline project_task_category/README.rst]: No need to change License

Comment (nikul-serpentcs) [inline project_task_category/README.rst:3]: no need to change License

**Review by nikul-serpentcs — CHANGES_REQUESTED** [review]
Minor Change

File: project_task_category/README.rst [hunk 3]
```diff
@@ -0,0 +1,42 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```
Comment (elicoidal) [inline project_task_category/README.rst:1]: switch to fragmented README

File: project_task_category/data/project_data.xml [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+    <data noupdate="1">
```
Comment (elicoidal) [inline project_task_category/data/project_data.xml:7]: merge with odoo

**Review by elicoidal — COMMENTED** [review]
almost there

**Comment by emagdalenaC2i** [comment]
I get this error when I try to test it in Runbot

Style error
×
The style compilation failed, see the error below. Your recent actions may be the cause, please try reverting the changes you made.

Error: Invalid CSS after "...    background:": expected expression (e.g. 1px, bold), was "@progress-bar-succe"
        on line 9683 of stdin
>>         background: @progress-bar-success-bg;
   --------------------^
This error occured while compiling the bundle 'web.assets_backend' containing:
    - /web/static/lib/bootstrap/scss/_functions.scss
    - /web/static/lib/bootstrap/scss/_mixins.scss
    - /web/static/src/scss/utils.scss
    - /web/static/src/scss/bs_mixins_overrides.scss
    - /web/static/src/scss/primary_variables.scss
    - /web_editor/static/src/scss/web_editor.variables.scss
    - /mail/static/src/scss/variables.scss
    - /portal/static/src/scss/primary_variables.scss
    - /web/static/src/scss/secondary_variables.scss
    - /web/static/src/scss/bootstrap_overridden.scss
    - /web_editor/static/src/scss/bootstrap_overridden.scss
    - /web/static/src/scss/import_bootstrap_part1.scss
    - /web/static/src/scss/bs_mixins_overrides.scss
    - /web/static/src/scss/import_bootstrap_part2.scss
    - /web/static/src/scss/bootstrap_review.scss
    - /web/static/src/scss/webclient_extra.scss
    - /web/static/src/scss/webclient_layout.scss
    - /web/static/src/scss/webclient.scss
    - /web/static/src/scss/domain_selector.scss
    - /web/static/src/scss/model_field_selector.scss
    - /web/static/src/scss/progress_bar.scss
    - /web/static/src/scss/dropdown.scss
    - /web/static/src/scss/dropdown_extra.scss
    - /web/static/src/scss/tooltip.scss
    - /web/static/src/scss/debug_manager.scss
    - /web/static/src/scss/control_panel.scss
    - /web/static/src/scss/fields.scss
    - /web/static/src/scss/fields_extra.scss
    - /web/static/src/scss/views.scss
    - /web/static/src/scss/pivot_view.scss
    - /web/static/src/scss/graph_view.scss
    - /web/static/src/scss/form_view.scss
    - /web/static/src/scss/form_view_extra.scss
    - /web/static/src/scss/list_view.scss
    - /web/static/src/scss/list_view_extra.scss
    - /web/static/src/scss/kanban_dashboard.scss
    - /web/static/src/scss/kanban_examples_dialog.scss
    - /web/static/src/scss/kanban_column_progressbar.scss
    - /web/static/src/scss/kanban_view.scss
    - /web/static/src/scss/kanban_view_mobile.scss
    - /web/static/src/scss/web_calendar.scss
    - /web/static/src/scss/search_view.scss
    - /web/static/src/scss/search_view_mobile.scss
    - /web/static/src/scss/dropdown_menu.scss
    - /web/static/src/scss/search_view_extra.scss
    - /web/static/src/scss/data_export.scss
    - /base/static/src/scss/onboarding.scss
    - /web/static/src/scss/base_settings.scss
    - /web/static/src/scss/report_backend.scss
    - /base_import/static/src/scss/base_import.scss
    - /web_diagram/static/src/scss/diagram_view.scss
    - /web_editor/static/src/scss/web_editor.common.scss
    - /web_editor/static/src/scss/web_editor.backend.scss
    - /mail/static/src/scss/announcement.scss
    - /mail/static/src/scss/discuss.scss
    - /mail/static/src/scss/abstract_thread_window.scss
    - /mail/static/src/scss/thread_window.scss
    - /mail/static/src/scss/composer.scss
    - /mail/static/src/scss/chatter.scss
    - /mail/static/src/scss/followers.scss
    - /mail/static/src/scss/thread.scss
    - /mail/static/src/scss/systray.scss
    - /mail/static/src/scss/mail_activity.scss
    - /mail/static/src/scss/activity_view.scss
    - /mail/static/src/scss/kanban_view.scss
    - /mail/static/src/scss/attachment_box.scss
    - /hr/static/src/scss/hr.scss
    - /partner_autocomplete/static/src/scss/partner_autocomplete.scss
    - /project/static/src/scss/project_dashboard.scss
    - /project_timeline/static/src/less/project_timeline.less
    - /project_timeline_hr_timesheet/static/src/scss/project_timeline_hr_timesheet.scss
    - /web_settings_dashboard/static/src/scss/dashboard.scss

**Comment by emagdalenaC2i** [comment]
@ruter-lyu Please, Can you Rebuild Runbot in order of fix in #496 ao I can review this module?

**Comment by nikul-serpentcs** [comment]
@ruter-lyu Rebase your branch

**Comment by ovnicraft** [comment]
PR #470 was already done, this must consider it to be compatible ?

**Comment by pedrobaeza** [comment]
Yes, this one is now meaningless.

**Pull request closed without merging**